src/organisms/subject-attend.tsx

```diff
@@ -0,0 +1,48 @@
+import { useEffect, useState } from 'react';
+import AttendanceStatus from '../molecules/attendance-status';
+import { API_ROOT } from "@/queries/config";
+import { useSearchParams } from "next/navigation";
+import styles from './subject-attend.module.css';;
+
+type AttendancesSum = {
+  onTime: number;
+  late: number;
+  miss: number;
+};
+
+const SubjectAttend = () => {
+    const queryParams = useSearchParams();
+    const subjectId = queryParams.get('subject_id');
+    const [attendance, setAttendance] = useState<AttendancesSum | null>(null);
+  
+    useEffect(() => {
+      const fetchSubjectData = async () => {
+        try {
+          const url = `${API_ROOT}/subjects/${subjectId}/all_attendances`;
+          const response = await fetch(url);
+          if (!response.ok) {
+              return;
+          }
+
+          const data = (await response.json()) as AttendancesSum;
+          setAttendance(data);
+        } catch (error) {
+          console.error('科目データの取得に失敗しました:', error);
+        }
+      };
+  
+      fetchSubjectData();
+    }, [subjectId]);
+  
+    if (!attendance) {
+      return <div className={styles.message}>科目データを読み込んでいます...</div>;
```

Comment:
```suggestion
      return <div className={`surface-variant-text body-medium ${styles.message}`}>科目データを読み込んでいます...</div>;
```
CSS でフォントサイズや色の指定は行わないでください. 代わりに, `surface-variant-text` や `body-medium` といった既存のスタイルを使用しましょう.